org.brlcad.BRL_CAD.yaml

```diff
@@ -0,0 +1,69 @@
+app-id: org.brlcad.BRL_CAD
+runtime: org.kde.Sdk
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: mged
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+cleanup:
+  - /share/doc
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: BRL-CAD
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      cflags: -Wno-error
+      cxxflags: -Wno-error
+    config-opts:
+      - -DBRLCAD_BUNDLED_LIBS=ON
+      - -DBUILD_STATIC_LIBS:BOOL=OFF
+      - -DBRLCAD_EXTRADOCS:BOOL=OFF
+      - -DBRLCAD_ENABLE_OSG:BOOL=ON
+      - -DBUILD_TESTING:BOOL=OFF
+    sources:
+      - type: git
+        url: https://github.com/BRL-CAD/brlcad.git
+        commit: dc7c0b221cd74c13a3a4a6c74c679bf91cc8da42
+        tag: rel-7-34-2
+        x-checker-data:
+          type: git
+          tag-pattern: ^rel-([\d-]+)$
+
+    # .desktop files is removed from /misc/debian after install, so manually install them.
```

Comment:
no they are not.

```
$ find . -name \*.desktop
./misc/debian/archer.desktop
./misc/debian/brlcad-db.desktop
./misc/debian/brlcad-doc-mged.desktop
./misc/debian/brlcad-doc.desktop
./misc/debian/mged.desktop
./misc/debian/rtwizard.desktop
```

Reply:
Yeah, I was looking at the build dir not the top source dir. I change it now.